packages/node/src/generators/application/application.spec.ts

```diff
@@ -13,7 +18,7 @@ describe('app', () => {
   let tree: Tree;
 
   beforeEach(() => {
-    tree = createTreeWithEmptyV1Workspace();
+    tree = createTreeWithEmptyWorkspace();
```

Comment:
Most of the changes here are due to updating away from using `workspace.json`. The only real change for this feature work is adding `webpack.config.js` checks in assertions.